src/status_im/contexts/wallet/send/routes/view.cljs

```diff
@@ -202,12 +202,15 @@
           :disabled-chain-ids disabled-from-chain-ids})
         show-routes? (not-empty sender-network-values)]
     (rn/use-effect
-     #(when (and active-screen? (> (count token-available-networks-for-suggested-routes) 0))
-        (fetch-routes input-value valid-input? 2000))
+     (fn []
+       (when (and active-screen?
+                  (> (count token-available-networks-for-suggested-routes) 0)
+                  (not lock-fetch-routes?))
+         (fetch-routes value valid-input? 2000)))
```

Comment:
btw what is 2000 here? perhaps it's better to put this in a named var, or instead use a map for the arguments so it's obvious of the uses without looking at the function

Reply:
It is an arbitrary time for debouncing-dispatch fetch routes event. Here https://github.com/status-im/status-mobile/pull/19100 we changed it to 2000, I can add it in a named var of course.